python/helpers/persistent_storage.py

```
@@ -0,0 +1,158 @@
+import os
+import shutil
+from dataclasses import dataclass
+from typing import Iterable, List, Dict, Tuple
```

Ruff (0.14.0)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)